downstream/titles/release-notes/topics/async-24-20250326.adoc

```diff
@@ -0,0 +1,57 @@
+[id="async-24-20250326"]
+
+= {PlatformNameShort} patch release March 26, 2025
+
+The following enhancements and bug fixes have been implemented in this release of {PlatformNameShort}.
+
+[cols="1a,3a", options="header"]
+|===
+| Release date | Component versions
+
+| March 26, 2025  | 
+* {ControllerNameStart} 4.6.20
+* {EDAcontroller} 1.0.7	             
+* {HubNameStart} 4.9.3 
+* {LightspeedShortName} 2.4.250225	 
+* {PlatformNameShort}-collection installer (online): 2.4-10
+* {PlatformNameShort}-collection installer (bundle): 2.4-10
+* Receptor devel v1.5.3
+|===
+
+CSV Versions in this release:
+
+* Namespace-scoped Bundle: aap-operator.v2.4.0-0.1742444794
+
+* Cluster-scoped Bundle: aap-operator.v2.4.0-0.1742445409
+
+== General
+
+The `ansible.controller` collection has been updated to 4.5.20.(AAP-42241)
```

Comment:
This is a super nit, but should there be a space before "(AAP-42241)"? I noticed that this is the convention throughout, so maybe it's ok as is. So feel free to ignore this question if it doesn't suit.

Reply:
Let me fix that. Thanks.

Reply:
It seems it just looks like that. There should be no spaces after the full stop. The rest look okay.